docs/hub/diffusers.md

```diff
@@ -0,0 +1,49 @@
+# Using 🧨 `diffusers` at Hugging Face
+
+🧨 `diffusers` is the go-to library for state-of-the-art pretrained diffusion models for generating images, audio, and even 3D structures of molecules. Whether you’re looking for a simple inference solution or want to train your own diffusion model, 🤗 Diffusers is a modular toolbox that supports both. Our library is designed with a focus on usability over performance, simple over easy, and customizability over abstractions.
```

Comment:
```suggestion
🧨 `diffusers` is the go-to library for state-of-the-art pretrained diffusion models for generating images, audio, and even 3D structures of molecules. Whether you’re looking for a simple inference solution or want to train your own diffusion model, 🤗 Diffusers is a modular toolbox that supports both. The library is designed with a focus on usability over performance, simple over easy, and customizability over abstractions.
```

Reply:
i would remove (most of) the emojis here too